### ISSUE TITLE
replaceData() does not work in 5.5.0 when table is not visible

### ISSUE THREAD

**Issue by andy-5** [olifolkerd/tabulator#4214]
**Describe the bug**
Loading new data with `table.replaceData(...)` does not work when the table is not visible (e.g. a parent container has `display: none` set). When the table becomes visible again, it will be empty. This used to work with e.g. Tabulator 5.4.4, but does not work with 5.5.0.

**Tabulator Info**
- Which version of Tabulator are you using?

  The described behaviour is seen with Tabulator 5.5.0.

**Working Example**
https://jsfiddle.net/n1f6ax0L/18/

**To Reproduce**
A step by step guide to recreate the issue in your JS Fiddle or Codepen:
1. Go to https://jsfiddle.net/n1f6ax0L/18/
2. Click on the 'Refresh' button
3. See empty table (even though it should have data)

**Expected behavior**
I expect the table to show the data loaded with `replaceData()`, as it was the case with Tabulator 5.4.4.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Firefox
 - Version: 113

**Comment by AmolPawar7181** [comment]
Hi @andy-5,

Can you try reorder the lines as below, this works with 5.5
 
```
//Sample Data
var tableData = [
	{name: "Bob"},
  {name: "Steve"},
  {name: "Jim"},
]

//Example Table
var table = new Tabulator("#table", {
  data: tableData, //load data into table
  height: 200, //enable the virtual DOM
  columns: [{
    title: "Name",
    field: "name",
    width: 200,
    editor: "input"
  }, ]
});

document.querySelector("button").addEventListener("click", () => {
  console.log("Data is cleared, container will be hidden")
  table.replaceData([]);
  document.querySelector("#container").style = "display:none;"
  setTimeout(() => {
    console.log("New data available, container will be shown");
    document.querySelector("#container").style = "display:initial;"
    table.replaceData(tableData);
    
  }, 0);
});

```

Can you please try this and see this works for you.
Thank you.

**Comment by andy-5** [comment]
Yes, in this simple example it is quite easy to fix. But in the larger application, where I discovered this, the visibility of the container and the data loading is triggered by different parts of the code base.

I implemented a similar workaround in my application, basically waiting for the DOM to update, but I think this issue can lead to quite some confusion and non-obvious errors depending on the timing of DOM updates and data updates (especially since it worked fine with previous Tabulator releases).
